src/index.html

```diff
@@ -7,8 +7,109 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:ital,wght@0,100;0,500;1,300&display=swap" 
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img 
+          src="images/logo.png" 
+          alt="#" 
+          class="img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_items">
+          <li class="nav__item">
+            <a
+              href="#" 
+              class="nav__link is-active"
+            >
+              <span class="text active">
+                APPLE
+              </span>
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link"
+            >
+              <span class="text">
+                SAMSUNG
+              </span>
+            </a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link"
+            >
+              <span class="text">
+                SMARTPHONES
+              </span>
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a 
              href="#" 
              class="nav__link"
            >
              <span class="text">
                SAMSUNG
              </span>
            </a>
          </li>

          <li class="nav__item">
            <a 
              href="#" 
              class="nav__link"
            >
              <span class="text">
                SMARTPHONES
              </span>
            </a>
          </li>
```

Add blank line between multiple sibling blocks